app/Http/Requests/CityAlternativeNameRequest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class CityAlternativeNameRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "name" => ["required", "string", "regex:/^[A-Z]/", "max:100", "unique:city_alternative_names"],
+            "city_id" => [],
```

Comment:
empty array?

Reply:
When "city_id", "country_id" or "providers" keys are missing in FormRequest they are also missing in Controller.

Reply:
Maybe `city_id` should have exist validation in the database or something

Reply:
Like that?
```php
"city_id" => ["exists:App\Models\City,id"]
```
or maybe

```php
"city_id" => ["exists:cities,id"]
```

Reply:
second option